index.html

```diff
@@ -138,16 +138,16 @@ <h3>Profesional Art Printing Data</h3>
                 </div>
             </div>
         </section>
-        <section class="about">
+        <section id="about" class="about">
             <div class="gen-desc">
                 <h2 class="heading-text">About me</h2>
-                <p class="paragraph-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="paragraph-text">Hello I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                 <button class="regular-btn">Get My Resume</button>
```

Comment:
- [ ] As suggested by the previous reviewer, the `Get M Resume` button does not have enough color contrast in the `hover` effect CSS property. I would kindly suggest you work on it and make sure the color contrast is well aligned with the requirement of 4.5:1

The color contrast result 👇 

![col-con](https://user-images.githubusercontent.com/78634841/214442025-9bdb45cd-bbb8-4c1b-b4c3-cff19250c2a3.png)

Reply:
Hello @brainconnect93, what tool are you using to measure the color contrast, so that I don't commit the same mistake again.

Reply:
@jlvFlores, kindly use this [tool](https://www.aditus.io/button-contrast-checker/) to check if you are good to go